server/server.js

```diff
@@ -77,9 +77,9 @@ module.exports = port => {
             currentUsers.push(user);
             gameLogic.joinGame(user);
             emitGameState();
-            res.sendStatus(200);
+            res.status(200).json({message: "OK"});
         } else {
-            res.status(404).json({message: "User already exists"});
+            res.status(200).json({message: "User already exists"});
```

Comment:
Maybe user alread logged in?

Reply:
Also, maybe it's best not to have a 200 response since technically shouldn't happen.